app/lib/shared/search_service.dart

```diff
@@ -243,17 +286,18 @@ class SearchQuery {
         params['platforms'] = platformPredicate.toQueryParamValue();
       }
     }
-    if (hasPackagePrefix) {
-      params['pkg-prefix'] = packagePrefix;
-    }
     if (order != null) {
       final String paramName = v2 ? 'sort' : 'order';
       params[paramName] = serializeSearchOrder(order);
     }
     if (page != null) {
       params['page'] = page.toString();
     }
-    return new Uri(path: path, queryParameters: params).toString();
+    if (params.isEmpty) {
```

Comment:
Isn't the empty params cased subsumed in the general case below?

Or is that a hanging `?`?

Reply:
Yeah, the hanging `?` is the issue. Either the map should become `null`, or returning the `path` directly .

Reply:
kk